src/pymortests/lyapunov.py

```diff
@@ -95,16 +126,47 @@ def _check_availability(lyap_solver):
 @pytest.mark.parametrize('m', m_list)
 @pytest.mark.parametrize('with_E', [False, True])
 @pytest.mark.parametrize('trans', [False, True])
-@pytest.mark.parametrize('n,lyap_solver', chain(product(n_list_small, lyap_dense_solver_list),
-                                                product(n_list_big, lyap_lrcf_solver_list)))
-def test_lrcf(n, m, with_E, trans, lyap_solver):
+@pytest.mark.parametrize('n,lyap_solver', chain(product(n_list_small, cont_lyap_dense_solver_list),
+                                                product(n_list_big, cont_lyap_lrcf_solver_list)))
+def test_cont_lrcf(n, m, with_E, trans, lyap_solver):
+    _check_availability(lyap_solver)
+
+    if not with_E:
+        A = conv_diff_1d_fd(n, 1, 0.1)
+        E = None
+    else:
+        A, E = conv_diff_1d_fem(n, 1, 0.1)
+    np.random.seed(0)
+    B = np.random.randn(n, m)
+    if trans:
+        B = B.T
+
+    Aop = NumpyMatrixOperator(A)
+    Eop = NumpyMatrixOperator(E) if with_E else None
+    Bva = Aop.source.from_numpy(B.T if not trans else B)
+
+    Zva = solve_cont_lyap_lrcf(Aop, Eop, Bva, trans=trans, options=lyap_solver)
+    assert len(Zva) <= n
+
+    Z = Zva.to_numpy().T
+    assert relative_residual(A, E, B, Z @ Z.T, trans=trans, cont_time=True) < 1e-10
+
+
+@pytest.mark.parametrize('n', n_list_small)
+@pytest.mark.parametrize('m', m_list)
+@pytest.mark.parametrize('with_E', [False, True])
+@pytest.mark.parametrize('trans', [False, True])
+@pytest.mark.parametrize('lyap_solver', disc_lyap_dense_solver_list)
+def test_disc_lrcf(n, m, with_E, trans, lyap_solver):
     _check_availability(lyap_solver)
 
     if not with_E:
         A = conv_diff_1d_fd(n, 1, 0.1)
         E = None
     else:
         A, E = conv_diff_1d_fem(n, 1, 0.1)
+
+    A /= np.max(np.abs(A)) * np.max(A.shape)  # ensure matrix is Schur-stable
```

Comment:
Since the matrices come from a semi-discretized PDE, I guess that a more natural way to generate Schur-stable matrices would be to use a fully discretized PDE, e.g., using explicit Euler time-stepping. In `conv_diff_1d_*` functions, you could add a `cont_time` argument, and if it is `False`, use `A = sps.eye(n) + dt * A` or `A = E + dt * A` (`dt = 0.1 / (4*a*(n+1)**2 + b*(n+1))` should work).